### ISSUE TITLE
Encoding issue for special characters on Chrome

### ISSUE THREAD

**Issue by baptiste-lacasse** [formatjs/formatjs#2839]
## Which package?
@formatjs/intl-displaynames

## Describe the bug
When using @formatjs/intl-displaynames version 4.0.8 or higher, there is an encoding issue when attempting to retrieve the name of a language that contains special characters on Chrome (version 89.0.4389.128 64-bit). 

## To Reproduce
Call `new Intl.DisplayNames(['fr'], {type: 'language'}).of('fr');`. 
Result in Firefox (87.0): "français"
Result in Chrome (89.0.4389.128): "franÃ§ais"

## Expected behavior
Expected result is "français" in all browsers.

## Screenshots
![image](https://user-images.githubusercontent.com/82821123/115245637-6f69d200-a125-11eb-91fd-7fab328ed85e.png)

## Desktop:

- OS: Windows 10
- Browser: Chrome 
- Version: 89.0.4389.128 64-bit

**Comment by longlho** [comment]
Do you have a codesandbox? Are you sure it's not a chrome issue?

**Comment by baptiste-lacasse** [comment]
After further investigation, the issue was caused by the fact that our JS file containing intl-displaynames was loaded with the default ISO-8859-1 character set. Prior to version 4.0.8 this did not seem to cause any issues. I cannot reproduce the issue in codesandbox as it seems the character encoding is handled differently there. We resolved our issue by setting the JS character set to UTF-8.
